GIT links audit plus style changes (#6473)
DFE-Digital/apply-for-teacher-training: app/views/content/terms_candidate.md

- gather or use data from our website
- use abusive language or act abusively towards anyone you come into contact with

## Applying for courses

### Submitting your initial application

You can apply for up to 3 courses.

### Apply again if your application does not lead to a place

You can apply for another course if your initial application does not lead to a place.

This might be because you:

* do not get an offer
* decline any offers
* withdraw your application

You can apply again to one course at a time as many times as you like.

[Chat online for help applying again](<%= t('get_into_teaching.url_online_chat') %>).

### Making changes to your application

> Once you submit your application it is sent to your teacher training providers straight away.

Author: ```suggestion
Once you submit your application it's sent to your teacher training providers straight away.
```

> Contact us within 5 working days at <%= bat_contact_mail_to %> if you need to change something.

Reviewer: Ah, so we're tackling a common issue here on the 5 day rule. Perhaps we can bundle in changes to the terms of use alongside this?

You can ask us to add courses (if you have not reached your limit of 3) or change your personal details.

We cannot change your course choice if your teacher training provider does not respond or your application was unsuccessful.

> However, you can apply again to a course of your choice.

Reviewer: NB business rules work and direction of travel.

Author: yep - just temporary changes these
